modules/gatk4/determinegermlinecontigploidy/main.nf

```diff
@@ -0,0 +1,48 @@
+process GATK4_DETERMINEGERMLINECONTIGPLOIDY {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::gatk4=4.2.6.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.2.6.1--py36hdfd78af_1' :
+        'quay.io/biocontainers/gatk4:4.2.6.1--py36hdfd78af_1'}"
+
+    input:
+    tuple val(meta), path(tsv)
+    // path  model
```

Comment:
seems like model may be a possible file? In that case it should be provided as input